src/main/resources/static/js/subscription.js

```diff
@@ -495,6 +494,51 @@ jQuery(document).ready(function () {
         table.responsive.recalc();
     });
 
+    function isEditAndDeleteButtonsDisabled(subscriptionOwner, subscriptionName, ldapEnabled) {
+        if (ldapEnabled == false) {
+            // LDAP is NOT activated
+            return false
+        }
+
+        // Check if subscriptionOwner is defined
+        var subscriptionOwnerDefined = isUserNameDefined(subscriptionOwner)
+        if (subscriptionOwnerDefined == false) {
+            // LDAP is NOT activated or is anonymous subscription
+            return false
+        }
+
+        // Check if current user is logged in
+        var currentUser = sessionStorage.getItem("currentUser");
+        var isCurrentUserLoggedIn = isUserNameDefined(currentUser)
+        if (isCurrentUserLoggedIn == false) {
+            // Current user is not logged in
+            console.log("User must be logged in to edit subscription: '" + subscriptionName + "'." )
+            return true
+        }
+
+        var isCurrentUserOwner = subscriptionOwner == currentUser
```

Comment:
I prefer "isSubscritpionOwner" or put "subscriptionOwner == currentUser" directly in if statement.
if (!(subscriptionOwner == currentUser)).

Reply:
or "isUserSubscriptionOwner"